### PR TITLE
ci: Adds unit test on run

### PR DESCRIPTION
**Pull request by jhechtf**
## Description

Adds CI Testing on PR 

## Screenshot(s) (if applicable, e.g. style updates)

image

## PR Readiness Checklist

* General
    * [x] Verified all changes are related to feature described above (no scope creep)
* Branch Stuff
    * [x] Checked target and base branch (should be feature branch merging INTO `main`)
    * [x] Merged latest `main` into feature branch (so it has latest code)
    * [x] Resolved any conflicts between feature branch and `main`
* Test Stuff
    * [x] Added unit tests to cover modified / additional code
* GitHub Stuff (on the right 👉)
    * [ ] Updated labels
    * [ ] Linked issue to which this PR relates
    * [ ] Added assignee
    * [ ] Added at least one (1) reviewer

### CODE REVIEW

File: .github/workflows/pull_requests.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (Wolven531) [inline .github/workflows/pull_requests.yml:17]: Let's update this to `16.x`, for newer Node

```suggestion
        node-version: [16.x]
```

**Review by Wolven531 — CHANGES_REQUESTED** [review]
One minor change requested

File: .github/workflows/pull_requests.yml (hunk 1 above)
Comment (jhechtf) [reply .github/workflows/pull_requests.yml:17]: Yeah sounds good, I wasn't sure what version was being used.

**Comment by jhechtf** [comment]
Updated code to use node 16.

Tests are passing.

image

**Review by Wolven531 — APPROVED** [review]
Lgtm, 👍🏻

**Pull request merged**